packages/server/test/unit/util/editors_spec.ts

```diff
@@ -67,37 +66,17 @@ describe('lib/util/editors', () => {
       sinon.restore()
     })
 
-    it('returns a list of editors on the user\'s system with an "On Computer" option prepended and an "Other" option appended', () => {
```

Comment:
No longer needed, we no long append this option in `editors.ts`, instead we do it elsewhere. I don't think this placeholder, which is here for the purpose of having a placeholder in the UI,  should be the responsibility of this server module. 

More info: https://github.com/cypress-io/cypress/pull/18932/files#r750277563